front/app/components/IdeaCard/Body/index.tsx

```diff
@@ -96,10 +97,8 @@ const Body = ({ idea }: Props) => {
           anonymous={idea.data.attributes.anonymous}
         />
         <Separator aria-hidden>&bull;</Separator>
-        {!isNilOrError(locale) && (
-          <TimeAgo>
-            {timeAgo(Date.parse(idea.data.attributes.created_at), locale)}
-          </TimeAgo>
+        {!isNilOrError(locale) && publishedAt && (
```

Comment:
When there is no publication date, for example because the input is in screening, no date is shown.